[2022-02-19 14:41:04,811] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: spark-test-cassandra-papermill.create_table scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-02-19 14:41:04,855] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: spark-test-cassandra-papermill.create_table scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-02-19 14:41:04,855] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 14:41:04,855] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 14:41:04,855] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 14:41:04,910] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): create_table> on 2020-01-01 00:00:00+00:00
[2022-02-19 14:41:04,921] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'spark-test-cassandra-papermill', 'create_table', 'scheduled__2020-01-01T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/cassandra_create_test_database.py', '--cfg-path', '/tmp/tmpalf8xt6k', '--error-file', '/tmp/tmp9da3zk5w']
[2022-02-19 14:41:04,916] {standard_task_runner.py:52} INFO - Started process 536 to run task
[2022-02-19 14:41:04,922] {standard_task_runner.py:77} INFO - Job 10: Subtask create_table
[2022-02-19 14:41:05,033] {logging_mixin.py:109} INFO - Running <TaskInstance: spark-test-cassandra-papermill.create_table scheduled__2020-01-01T00:00:00+00:00 [running]> on host 38342be44bf7
[2022-02-19 14:41:05,154] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=spark-test-cassandra-papermill
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T00:00:00+00:00
[2022-02-19 14:41:05,171] {base.py:79} INFO - Using connection to: id: ***_default. Host: 172.20.0.11, Port: 9042, Schema: None, Login: ***, Password: ***, extra: {}
[2022-02-19 14:41:05,173] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/apache/***/hooks/***.py:123: DeprecationWarning: Legacy execution parameters will be removed in 4.0. Consider using execution profiles.
  self.cluster = Cluster(**conn_config)

[2022-02-19 14:41:05,193] {cassandra.py:131} WARNING - Downgrading core protocol version from 66 to 65 for 172.20.0.11:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/***/cluster.html#***.cluster.Cluster.protocol_version
[2022-02-19 14:41:05,213] {cassandra.py:131} WARNING - Downgrading core protocol version from 65 to 5 for 172.20.0.11:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/***/cluster.html#***.cluster.Cluster.protocol_version
[2022-02-19 14:41:05,219] {libevreactor.py:370} ERROR - Closing connection <LibevConnection(139645919694096) 172.20.0.11:9042> due to protocol error: Error from server: code=000a [Protocol error] message="Beta version of the protocol used (5/v5-beta), but USE_BETA flag is unset"
[2022-02-19 14:41:05,223] {cassandra.py:131} WARNING - Downgrading core protocol version from 5 to 4 for 172.20.0.11:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/***/cluster.html#***.cluster.Cluster.protocol_version
[2022-02-19 14:41:06,082] {cassandra_create_test_database.py:35} INFO - >>>>>>>>>>> Executed: Keyspace and Tables are created
[2022-02-19 14:41:06,083] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 14:41:06,154] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=spark-test-***-papermill, task_id=create_table, execution_date=20200101T000000, start_date=20220219T144104, end_date=20220219T144106
[2022-02-19 14:41:06,241] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 14:41:06,335] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
